cli/functionary/tokens.py

```diff
@@ -1,58 +1,25 @@
 import json
-from pathlib import Path
 
+import click
 import requests
-from dotenv import dotenv_values
-
-
-class TokenError(Exception):
-    pass
 
 
 def login(login_url: str, user: str, password: str):
-    message = "Login successful!"
-    success = False
 
     try:
         login_response = requests.post(
             f"{login_url}", data={"username": user, "password": password}
         )
         # check status code/message on return then exit
         if login_response.ok:
-            tokens = json.loads(login_response.text)
-            saveTokens(tokens)
-            success = True
+            token = json.loads(login_response.text).get("token")
+            return token
         else:
-            message = (
+            raise click.ClickException(
                 f"Failed to login: {login_response.status_code}\n"
-                f"\tResponse: {login_response.text}"
+                + f"Response: {login_response.text}"
```

Comment:
Super minor thing, but you don't need the `+` here.  The strings will get concatenated.